_guide/plain-language.md

```diff
@@ -17,112 +17,73 @@ roles:
-* Filler words. For example, use the word 'analyze' not 'conduct an analysis'.
-* Repetitive words 
-* Irony or idioms
-* Intro text. For example, "Welcome to our department homepage."
+Always run your content through a readability tool and try to get the reading grade level lower than 8th grade. Some common readability tools are:
 
-Try:
+* [Hemingway App](https://www.hemingwayapp.com/): This app highlights content that could be hard for your audience to read. 
+* [Readable](https://readable.io/): This is a paid service that analyzes your content to identify its readability score, grammar errors, and style issues. 
+* [Microsoft's Flesch-Kincaid](https://support.microsoft.com/en-us/office/get-your-document-s-readability-and-level-statistics-85b4969e-e80a-4777-8dd3-f7fc3c8b3fd2): This can be used with Microsoft Word to calculate your content's readability score. 
 
-* Using active voice throughout your site 
-* Writing one idea per sentence
-* Using the simpler synonym. For example, 'use' not 'utilize'.
-* Keeping the reading level between 6th and 8th grade reading level. Remember to remove proper nouns and titles.
+### Use a text-to-speech reader
+Text-to-speech (TTS) is a type of assistive technology that takes words on a digital device and reads it aloud. Simply hearing text read aloud can help authors better understand where improvements can be made. We recommend this tool:
 
-### Write for your audience 
+* [Microsoft's Immersive Reader](https://support.microsoft.com/en-us/topic/use-immersive-reader-in-microsoft-edge-78a7a17d-52e1-47ee-b0ac-eff8539015e1): In Microsoft Edge, this mode has a Read Aloud tool that can be used to read the text of a web page.
 
-Before writing, ask yourself: What are your reader's needs? What do they need to know or do after reading your content?
+### User testing with your audience
+Automated readability tools won't catch everything. You'll need to test your content with humans too. When user testing, you can: 
 
-### Make it actionable
+* Use tree testing to discover if menu labels and other content is easy to find. This is a research method where you ask users to look for important resources.
+* Ask your users to read your content aloud. This can help you identify what words they may struggle with. Users can also rate the clarity of the language. 
+* Ask your users to paraphrase what they read to gauge their understanding. 
 
-People visit your website to do something: answer a question or complete a specific task. Ask yourself: what are the top questions my target reader wants to know? How can I answer it quickly?
+## Resources
 
-### Choose words your reader uses
+* [Federal Plain Language guidelines](https://www.plainlanguage.gov/guidelines/)
+* [Plain Language Association International guidelines](https://plainlanguagenetwork.org/plain-language/what-is-plain-language/)
+* [WCAG 2.1 Reading level success criterion ](https://www.w3.org/WAI/WCAG21/Understanding/reading-level.html)
```

Comment:
@thejordanwood looks like an extra space here before the `]`

Reply:
Fixed! Thanks